src/SearchResults.js

```diff
@@ -14,14 +15,29 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
+  // eslint-disable-next-line camelcase
+  const page_size = 5;
+  const [
+    searchOptionsForSearchResults,
+    setSearchOptionsForSearchResults,
+  ] = useState({
+    ...SOUND_LIST_QUERY_PARAMS,
+    page_size,
+  });
   const [searchResults, setSearchResults] = useState([]);
+  const [totalNumberOfSearchResults, setTotalNumberOfSearchResults] = useState(
```

Comment:
naming suggestions: `resultsCount` or `numResults`. Search results is the name of the component so that implies these are search results.

Reply:
b1cd80b addresses this. I don't think there are any further changes to make concerning this so I'll resolve this conversation.